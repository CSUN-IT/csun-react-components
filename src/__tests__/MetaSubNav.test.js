import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import MetaSubNav from '../../src/components/meta/SubNav';

configure({ adapter: new Adapter() });

describe('MetaSubNav', () => {
  it('Should be defined', () => {
    expect(MetaSubNav).toBeDefined();
  });

  it('Renders component correctly', () => {
    const tree = shallow(<MetaSubNav />);
    expect(tree).toMatchSnapshot();
  });
});
