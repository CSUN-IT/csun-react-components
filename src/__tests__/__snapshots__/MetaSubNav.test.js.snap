// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetaSubNav Renders component correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MetaSubNav
    className=""
    menuItems={Array []}
    typeColor=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul
        className="tabs cf"
      />,
      "className": " tab-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [],
        "className": "tabs cf",
      },
      "ref": null,
      "rendered": Array [],
      "type": "ul",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul
          className="tabs cf"
        />,
        "className": " tab-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
          "className": "tabs cf",
        },
        "ref": null,
        "rendered": Array [],
        "type": "ul",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
